docs/source/posts/2022/2022-09-28-week-14-shivam.rst

```diff
@@ -0,0 +1,45 @@
+Week 14 - Morphing is here! 
+===========================
+
+.. post:: September 28 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week, I started with Multiple actor support in skinning, Managed to do it
+  successfully (see brainStem Model)
+
+- Implementing multiple animation channels support (as seen in the ``Fox`` model). The ``get_skin_timelines()`` method now returns a dictionary of all animation channels with Timeline as their value.
+
+- We merged two PRs, `#689`_ (colors from Material) and `#643`_ (simple animations).
+
+- Added ability to load morphing information and create timelines from it. Here's a preview of the ``AnimatedMorphCube`` and ``AnimatedMorphSphere``models:
+
+    .. raw:: html
+
+        <iframe id="player" type="text/html"   width="1280" height="720" src="https://user-images.githubusercontent.com/74976752/192871376-881fbdd6-2fab-4a7f-9f4f-07663d93561c.mp4" frameborder="0"></iframe>
```

Comment:
The video is overlapping the right side legends.

![msedge_1Rk3a1jvB8](https://user-images.githubusercontent.com/64432063/193049149-71d5c77e-2a0e-464d-94dc-b7f558c6a095.png)